dom0/sd-workstation-template-files.sls

```diff
@@ -11,6 +11,14 @@ sd-workstation-template-install-kernel-config-packages:
     - require:
       - sls: fpf-apt-test-repo
 
+
+sd-workstation-install-securedrop-log-package:
```

Comment:
All other templates are cloned from this template, but in an in-place scenario, the template would not get upgraded. One would need to make clean and make all for these changes to propagate to all the other sd-* templates. I think the approach makes sense here, install in the base template, and also install it vm-specific states